dll/win32/browseui/lang/fr-FR.rc

```diff
@@ -354,3 +354,9 @@ BEGIN
     IDS_COL_LOCATION "Dans le dossier"
     IDS_COL_RELEVANCE "Pertinence"
 END
+
+STRINGTABLE
+BEGIN
+    IDS_PARSE_ADDR_ERR_TITLE "Explorateur ReactOS"
```

Comment:
Current windows text is "Explorateur de fichiers"

Reply:
We decided above to use 'ReactOS Explorer', which would translate to 'L'explorateur ReactOS' (commend from HBelusca above).